letsencrypt-auto-source/pieces/fetch.py

```diff
@@ -70,6 +70,10 @@ def write(contents, dir, filename):
     with open(join(dir, filename), 'w') as file:
         file.write(contents)
 
+def writebytes(contents, dir, filename):
```

Comment:
I would try to eliminate the repetition between this and write(). We could just add an `is_bytes` kwarg to write(), for example.